### DISCUSSION TITLE
关于中文输出乱码的问题

### DISCUSSION THREAD

**Discussion by constly** [ejoy/ant#51] [General]
系统windows10，默认编码936。
我分别用cmd 和 powershell启动测试工程，在print(“中文输出”)时，控制台均显示乱码，
同样的环境，比如我执行nodejs程序，输出中文却是正常的。

后面我尝试先改控制台编码，再启动程序，中文显示正常，批处理如下：
```
chcp 65001
"bin/msvc/release/lua.exe" "game/entry.lua"
```
不知道方不方便在代码层面做下兼容处理~

**Comment by cloudwu** [comment]
这个输出不是乱码，是 utf8 的字符串。
所以解决方法是使用支持 utf8 显示的环境吧？

**Reply by constly** [reply]
确实，不过理论上说，应该有两个解决办法：
1. 让显示环境支持utf8
2. 检查显示环境，把utf8转换为对应的编码

**Reply by cloudwu** [reply]
不考虑 2 , 没必要把开发精力花在功能不健全的控制台上。
